.travis.yml

```diff
@@ -1,5 +1,5 @@
-language:
```

Comment:
Just rebase this branch on development, so you will not need to add this manually.